crates/server/src/server/metrics.rs

```diff
@@ -0,0 +1,273 @@
+use hickory_proto::op::{Header, OpCode, ResponseCode};
+use hickory_proto::xfer::Protocol;
+use metrics::{Counter, Unit, counter, describe_counter};
+
+#[derive(Clone)]
+pub(super) struct ReportingResponseHandlerMetrics {
```

Comment:
Remove `Reporting` from this name? It doesn't seem to add much.

Reply:
Dropped that part.